corehq/apps/accounting/forms.py

```diff
@@ -682,6 +682,13 @@ def clean_active_accounts(self):
         if transfer_account and transfer_account == self.subscription.account.id:
             raise ValidationError(_("Please select an account other than the "
                                     "current account to transfer to."))
+        if transfer_account:
+            acct = BillingAccount.objects.get(id=transfer_account)
+            if acct.is_customer_billing_account != self.subscription.plan_version.plan.is_customer_software_plan:
+                if acct.is_customer_billing_account:
+                    raise ValidationError(mark_safe(_("Please select a regular Billing Account to transfer to.")))
```

Comment:
We don't need to translate this text, because it's an admin-only page.  While it's important to mark all user-facing strings for translation, it's good to skip marking other strings so that way the people doing translations don't waste their time.

(The accounting admin pages are inconsistent about following this, but we should strive towards fixing that.)